### PR TITLE
Create Generic disbot.js

### PR DESCRIPTION
**Pull request by aelagrassa**
This is a generic disbot file that has many of the previous media/audio files removed. It's only real feature is using YTDL and showing a few examples of responding to text and playing a single mp3 file [horn].

### CODE REVIEW

File: disbot.js [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+        voice_channels.push(channel);
+      }
+    }
+
+    // for each @mention, move them around X times
+    for(var mem_index in members){
+      var member = members[mem_index];
+    }
+  }
+
```
Comment (timm3) [inline disbot.js:192]:
```suggestion
```

File: disbot.js [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+  /*
+    Def: kill process
+    Args:
+    Notes:
+  */
+  function cease(){
+    console.log('!cease performed');
+    process.exit();
+  }
```
Comment (timm3) [inline disbot.js:159]:
```suggestion
```

File: disbot.js [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+        console.log(err);
+      }
+      ret = 'file dl attempted?'
+    } else {
+      var ret = 'no file found';
+    }
+    console.log(ret);
+    return ret;
+  }
+
```
Comment (timm3) [inline disbot.js:106]:
```suggestion
```

File: disbot.js [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+            var maybe_roll_max = parseInt(args.pop());
+            var roll_max = isNaN(maybe_roll_max) ? 100 : maybe_roll_max;
+            roll(message, roll_max);
+            break;
+          case '!yt':
+            var link = args.shift();
+            yt(voice_channel, link, username, message);
+            break;
+          /* fall-through */
+          case '/join':
```
Comment (timm3) [inline disbot.js:278]:
```suggestion
```

Comment (timm3) [reply disbot.js:278]: `/join` is handled in the other message handler, and should not cause the bot to say that the bot is stopping

File: disbot.js [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+          case '!stop':
+            message.reply('Stopping boss!');
+            break;
```
Comment (timm3) [inline disbot.js:281]:
```suggestion
```

I'd handle the `!stop` command in just one location, and move `message.reply('Stopping boss!');` to the other handler for this special command

**Review by timm3 — CHANGES_REQUESTED** [review]
This stripped down bot is a good start, and just needs to be stripped down a little further to be a base/starter/generic bot.
Please remove the dead code and consolidate the handling of `/join` and `!stop` commands.

**Comment by aelagrassa** [comment]
Closing due to noobery

**Pull request closed without merging**